Example/Example/CharactersPresenter.swift

```diff
@@ -17,6 +17,15 @@ class CharactersPresenter: BothamPresenter {
     }
 
     func viewDidLoad() {
-        self.ui?.showItems([Character(name: "Spiderman"), Character(name: "IronMan")])
+        loadItems([Character(name: "Spiderman"), Character(name: "IronMan")])
+    }
+
+    func refresh(didRefresh: () -> ()) {
+        loadItems([Character(name: "Captain America"), Character(name: "Magneto"), Character(name: "Star-Lord")])
+        didRefresh()
```

Comment:
We could replace this call with a ui?.stopRefreshing() method. Doing this you don't have to start connecting your callbacks.